docs/user_guides/integrations/beam.md

```diff
@@ -0,0 +1,47 @@
+---
+description: Documentation on how to configure an Apache Beam Dataflow Runner job to write features in to the Hopsworks Feature Store
+---
+# Apache Beam Dataflow Runner
+
+Connecting to the Feature Store from an Apache Beam Dataflow Runner, requires Hopsworks certificates. For this in your Beam Java application `pom.xml` file include following snippet:
+```
+    <resources>
+      <resource>
+        <directory>java.io.tmpdir</directory>
+        <includes>
+          <include>**/*.jks</include>
+        </includes>
+      </resource>
+    </resources>
```

Comment:
I take it that one should download the certificates first and then put them in the /tmp dir to be packaged it with the jar file?

If so, the documentation doesn't mention that you need to download the files and I would explicitly  put in the snippet the `keyStore.jks` and `trustStore.jks` - Also what about th e password file?

Reply:
You don't need to download certificates manually https://github.com/logicalclocks/feature-store-api/blob/5e78563e6a7e2bfe17b6ceb6c804a5d66be96f0b/java/hsfs/src/main/java/com/logicalclocks/hsfs/metadata/HopsworksClient.java#L121

That is downloaded during the execution of `mvn compile exec:java`